sdk/transaction_model.go

```diff
@@ -2903,8 +2906,11 @@ const (
 	ReplicatorOffboarding          EntityType = 0x4762
 	CreateLiquidityProvider        EntityType = 0x4169
 	ManualRateChange               EntityType = 0x4269
-	PlaceSdaExchangeOffer     EntityType = 0x416A
-	RemoveSdaExchangeOffer    EntityType = 0x426A
+	PlaceSdaExchangeOffer     	   EntityType = 0x416A
+	RemoveSdaExchangeOffer    	   EntityType = 0x426A
+	DeployContract				   EntityType = 0x416E
+	ManualCall					   EntityType = 0x426E
+	AutomaticExecutionsPayment	   EntityType = 0x436E
```

Comment:
formatting

Reply:
and add types and versions for `Successful and Unsuccessful EndBatchExecutionTransaction`